packages/sanes-chrome-extension/src/routes/recovery-phrase/utils/pdfGenerator.ts

```diff
@@ -26,6 +17,17 @@ class PDFGenerator {
   ];
 
   public constructor() {
+    // Loading the module 'jspdf' in an environment with no canvas available leads to the error message
+    // > Error: Not implemented: HTMLCanvasElement.prototype.getContext
+    // See also https://github.com/iov-one/ponferrada/issues/133
+    // To avoid this error message to pollute test outputs, we load jspdf only when PDFGenerator is
+    // constructed, not when it is imported.
+
+    // eslint-disable-next-line @typescript-eslint/no-var-requires
+    const { jsPDF } = require('jspdf');
```

Comment:
this import is probably wrong (`const jsPDF = require('jspdf');`) but I could not read CI error message before it was restared

Reply:
No, there is an issue with Travis from time to time, it can not pass one E2E test, it failed by timeout (maybe because of lack of resources, I do not know). But restart helps it to finish. So please do not worry. At least I hope this will help.

Reply:
Hmm, it doesn't helps.

Reply:
Let me try to reproduce locally. The test did not run locally before since I did not start the blockchains

Reply:
After researching a bit, I believe this should be:
`const jsPDF = require('jspdf');`

But it would not surprise me if using `cont { jsPDF } =` works as well.

Reply:
In my case (I got this fail some time ago) locally tests was running properly, but it was failed in Travis, and as I said restart was helpws to fix this.

Reply:
`const jsPDF = require('jspdf');` is correct, tested locally. PR was updated